src/main/java/hlf/java/rest/client/service/impl/NetworkStatusImpl.java

```diff
@@ -288,25 +316,63 @@ public ResponseEntity<ClientResponseModel> commitChannelConfigTransaction(
 
   @Override
   public ResponseEntity<ClientResponseModel> addOrgToChannel(
-      String channelName, NewOrgParamsDTO organizationDetails) {
+      String channelName, ChannelUpdateParamsDTO organizationDetails) {
     Network network = gateway.getNetwork(channelName);
     if (network != null && user != null) {
       try {
         Channel selectedChannel = network.getChannel();
-        ConfigUpdate selectedChannelConfigUpdate =
-            ConfigUpdate.parseFrom(selectedChannel.getChannelConfigurationBytes());
-        // ReadSet must have version, all other fields ignored; WriteSet must have changes
-        ConfigGroup readset = selectedChannelConfigUpdate.getReadSet();
-        // ConfigGroups consist of: groups, modPolicy, policies, values, and version.
-        ConfigUpdate configUpdate =
-            ConfigUpdate.newBuilder()
-                .setChannelId(channelName)
-                .setReadSet(readset)
-                .setWriteSet(
-                    addOrgToChannelWriteSetBuilder.buildWriteset(readset, organizationDetails))
-                .build();
-        MessageOrBuilder message = configUpdate;
-        String channelConfigString = JsonFormat.printer().print(message);
+        ConfigUpdate configUpdate = createConfigUpdate(channelName, organizationDetails);
+        String channelConfigString = JsonFormat.printer().print(configUpdate);
```

Comment:
[question] The method `getDeserializedConfig` appears to be doing this task with the exception handling.

Reply:
The existing logic was to print it as string and use it further, kept the logic as in. The `getDeserializedConfig` method returns the Base64 encoded characters.